lib/realtime/api.ex

```diff
@@ -124,7 +124,7 @@ defmodule Realtime.Api do
          data: %{external_id: external_id}
        })
        when is_map_key(changes, :jwt_jwks) or is_map_key(changes, :jwt_secret) do
-    Phoenix.PubSub.broadcast!(Realtime.PubSub, "realtime:operations:" <> external_id, :disconnect)
+    Phoenix.PubSub.broadcast!(Realtime.PubSub, "user_socket:" <> external_id, "disconnect")
```

Comment:
This works?

Reply:
this has changed to something different but I can't test the socket closing but I do get a call on the terminate callback of RealtimeChannel... 

still the websocket process is still alive in the integrations tests so I'm not fully sure on how to properly tackle it

also we only receive the terminate meaning we can't inform the user that the channeil is going to close due to server request

Ppushed latest to this PR